vignettes/ms2deepscore/tutorial.Rmd

```diff
@@ -26,7 +26,7 @@ You can also select any other from the [GNPS spectral library](https://gnps-exte
 
 ```{r}
 # Load your own MS/MS dataset
-path_to_mgf <- '../../notebook/ms2deepscore/GNPS-2-SPECTRA.mgf'
+path_to_mgf <- 'GNPS-2-SPECTRA.mgf'
```

Comment:
I think this path should stay relative no?

Reply:
That is strange. I did only convert them :O

Reply:
yes, it's normal. and in the notebook we only have the 'GNPS-2-SPECTRA.mgf' because we are in the right dir